style.css

```diff
@@ -0,0 +1,104 @@
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+  font-family: Arial, Helvetica, sans-serif;
+}
+
+html {
+  scroll-behavior: smooth;
+}
+
+.mobile {
+  display: none;
+}
```

Comment:
- [ ] It would be great if you write README.md file more descriptive. For doing this, you need to follow [this template](https://github.com/microverseinc/readme-template). 

[SUGGESTION] You can copy and past [the given template](https://github.com/microverseinc/readme-template) and personalize it. 😊